Inheritance/src/Inheritance/ActorExtention.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Inheritance {
+    public static class ActorExtention {
+        public static String Speak(Actor actor) {
```

Comment:
```suggestion
        public static String Speak(this Actor actor) {
``` 
the extention method should be called like this?

Reply:
I don't believe so, since I pass in an actor. I tried changing it as suggested and it made no difference. Is there a reason to that I've forgotten?

Reply:
With how you currently have it, Speak() is not an extension method for the Actor class, only a static method in the ActorExtention static class. For example, you can call ActorExtention.Speak(), but an object of type Actor cannot call Speak() directly. If you put "this" at the beginning of the parameter as suggested, Speak() now becomes an extension method for the Actor class, and you should be able to call ActorExtention.Speak(), as well as call Speak() directly from any Actor object and all child classes of Actor.

Reply:
Gotcha, that makes a lot of sense. Thanks for explaining!